Sources/Files.swift

```diff
@@ -485,6 +485,21 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public var subfolders: FileSystemSequence<Folder> {
         return makeSubfolderSequence()
     }
+
+    /// A reference to the folder that is the current working directory
+    public static var current: Folder {
+        return FileSystem(using: .default).currentFolder
```

Comment:
Don't use `.default`, use the Folder's `fileManager`. That way, if a custom one is used, it's not lost.